### PR TITLE
[zh-cn]: update the translation of `mathml`

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
#20746

### CODE REVIEW

File: files/zh-cn/web/mathml/index.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+l10n:
+  sourceCommit: 6e0504e4a750ac4f495086c7aec3bfd293ad6417
 ---
 
 {{MathMLRef}}
 
 **Mathematical Markup Language (MathML)** 是一个用于描述数学公式、符号的一种 [XML](/zh-CN/docs/XML) 标记语言。
 
-在这里你可以找到有关的文档、例子和工具来帮助你使用这个强大的技术。如果想快速了解这项技术，请看 [Slides for the innovation fairs at Mozilla Summit 2013](http://fred-wang.github.io/MozSummitMathML/index.html)。
+[MathML](https://w3c.github.io/mathml/) 最初被设计为用于浏览器、办公套件、[计算机代数系统](https://zh.wikipedia.org/wiki/計算機代數系統)、[EPUB](https://www.w3.org/publishing/epub33/)阅读器和基于[LaTeX](https://zh.wikipedia.org/wiki/LaTeX)的生成器的通用规范。然而，这种方法并不非常适用于 Web：[专注于语义的子集](https://w3c.github.io/mathml/#contm)从未在浏览器中实现，而[专注于数学布局](https://w3c.github.io/mathml/#presm)的子集导致了不完整和不一致的浏览器实现。
```
Comment (T34-active) [inline files/zh-cn/web/mathml/index.md:12]:
```suggestion
[MathML](https://w3c.github.io/mathml/) 最初被设计为用于浏览器、办公套件、[计算机代数系统](https://zh.wikipedia.org/wiki/計算機代數系統)、[EPUB](https://www.w3.org/publishing/epub33/) 阅读器和基于 [LaTeX](https://zh.wikipedia.org/wiki/LaTeX) 的生成器的通用规范。然而，这种方法并不非常适用于 Web：[专注于语义的子集](https://w3c.github.io/mathml/#contm)从未在浏览器中实现，而[专注于数学布局](https://w3c.github.io/mathml/#presm)的子集导致了不完整和不一致的浏览器实现。
```

File: files/zh-cn/web/mathml/index.md [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+
+[MathML Core](https://w3c.github.io/mathml-core/) 是一个基于[LaTeX](https://zh.wikipedia.org/wiki/LaTeX)和[开放字体格式](https://docs.microsoft.com/typography/opentype/spec/math)规则的增加了实现细节的子集。它专门为浏览器量身定制，并设计成与其他 Web 标准（包括[HTML](/zh-CN/docs/Web/HTML)、[CSS](/zh-CN/docs/Web/CSS)、[DOM](/zh-CN/docs/Web/API/Document_Object_Model)、[JavaScript](/zh-CN/docs/Web/JavaScript)）良好配合。
```
Comment (T34-active) [inline files/zh-cn/web/mathml/index.md:14]:
```suggestion
[MathML Core](https://w3c.github.io/mathml-core/) 是一个基于 [LaTeX](https://zh.wikipedia.org/wiki/LaTeX) 和[开放字体格式](https://docs.microsoft.com/typography/opentype/spec/math)规则的增加了实现细节的子集。它专门为浏览器量身定制，并设计成与其他 Web 标准（包括 [HTML](/zh-CN/docs/Web/HTML)、[CSS](/zh-CN/docs/Web/CSS)、[DOM](/zh-CN/docs/Web/API/Document_Object_Model)、[JavaScript](/zh-CN/docs/Web/JavaScript)）良好配合。
```

File: files/zh-cn/web/mathml/index.md [hunk 3]
```diff
@@ -1,13 +1,21 @@
...
+
+以下是有关 MathML 文档、示例和工具的链接。MDN 将[MathML Core](https://w3c.github.io/mathml-core/)用作参考规范，但由于不稳定的标准化历史，旧版 MathML 功能可能仍会出现在现有的实现和 Web 内容中。
```
Comment (T34-active) [inline files/zh-cn/web/mathml/index.md:16]:
```suggestion
以下是有关 MathML 文档、示例和工具的链接。MDN 将 [MathML Core](https://w3c.github.io/mathml-core/) 用作参考规范，但由于不稳定的标准化历史，旧版 MathML 功能可能仍会出现在现有的实现和 Web 内容中。
```

File: files/zh-cn/web/mathml/index.md [hunk 4]
```diff
@@ -1,13 +1,21 @@
...
+
+> **备注：** 强烈建议开发人员和作者切换到 MathML Core，或许依赖其他 Web 技术来满足缺失的用例。Math WG 正在维护一组[MathML polyfills](https://github.com/mathml-refresh/mathml-polyfills)以便促进这一过渡。
```
Comment (T34-active) [inline files/zh-cn/web/mathml/index.md:18]:
```suggestion
> **备注：** 强烈建议开发人员和作者切换到 MathML Core，或许依赖其他 Web 技术来满足缺失的用例。Math WG 正在维护一组 [MathML polyfill](https://github.com/mathml-refresh/mathml-polyfills) 以便促进这一过渡。
```

File: files/zh-cn/web/mathml/index.md [hunk 5]
```diff
@@ -19,24 +27,21 @@ slug: Web/MathML
   - : 可以让你理解如何使用 MathML 的例子。
 - [编写 MathML](/zh-CN/docs/Web/MathML/Authoring)
   - : 编写 MathML 的一些建议和提示，包括建议使用的编辑器以及如何将编辑器的输出集成到网页中去。
+- [MathML 教程](/zh-CN/docs/Learn/MathML)
+  - : 一个对 MathML 的简要介绍。
 
 [查看全部...](/zh-CN/docs/tag/MathML)
```
Comment (PassionPenguin) [inline files/zh-cn/web/mathml/index.md:33]:
```suggestion
```

File: files/zh-cn/web/mathml/index.md [hunk 6]
```diff
@@ -19,24 +27,21 @@ slug: Web/MathML
...
 
 ## 从社区获得帮助
 
-- [IRC 频道](irc://irc.mozilla.org/%23mathml)
-- [Mozilla 贡献者的 Wiki](https://wiki.mozilla.org/MathML:Home_Page)
 - [W3C Math 主页](https://www.w3.org/Math/)
+- [在 GitHub w3c/mathml 代码库中提出 issue](https://github.com/w3c/mathml/issues)
```
Comment (PassionPenguin) [inline files/zh-cn/web/mathml/index.md:38]:
```suggestion
- [在 GitHub w3c/mathml 代码库中提起议题](https://github.com/w3c/mathml/issues)
```

**Review by PassionPenguin — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/mathml/index.md [hunk 7]
```diff
@@ -1,13 +1,21 @@
...
 {{MathMLRef}}
 
 **Mathematical Markup Language (MathML)** 是一个用于描述数学公式、符号的一种 [XML](/zh-CN/docs/XML) 标记语言。
 
-在这里你可以找到有关的文档、例子和工具来帮助你使用这个强大的技术。如果想快速了解这项技术，请看 [Slides for the innovation fairs at Mozilla Summit 2013](http://fred-wang.github.io/MozSummitMathML/index.html)。
+[MathML](https://w3c.github.io/mathml/) 最初被设计为用于浏览器、办公套件、[计算机代数系统](https://zh.wikipedia.org/wiki/計算機代數系統)、[EPUB](https://www.w3.org/publishing/epub33/) 阅读器和基于 [LaTeX](https://zh.wikipedia.org/wiki/LaTeX) 的生成器的通用规范。然而，这种方法并不非常适用于 Web：[专注于语义的子集](https://w3c.github.io/mathml/#contm)从未在浏览器中实现，而[专注于数学布局](https://w3c.github.io/mathml/#presm)的子集导致了不完整和不一致的浏览器实现。
+
+[MathML Core](https://w3c.github.io/mathml-core/) 是一个基于 [LaTeX](https://zh.wikipedia.org/wiki/LaTeX) 和[开放字体格式](https://docs.microsoft.com/typography/opentype/spec/math)规则的增加了实现细节的子集。它专门为浏览器量身定制，并设计成与其他 Web 标准（包括 [HTML](/zh-CN/docs/Web/HTML)、[CSS](/zh-CN/docs/Web/CSS)、[DOM](/zh-CN/docs/Web/API/Document_Object_Model)、[JavaScript](/zh-CN/docs/Web/JavaScript)）良好配合。
+
+以下是有关 MathML 文档、示例和工具的链接。MDN 将 [MathML Core](https://w3c.github.io/mathml-core/) 用作参考规范，但由于不稳定的标准化历史，旧版 MathML 功能可能仍会出现在现有的实现和 Web 内容中。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/index.md:16]:
```suggestion
以下是有关 MathML 文档、示例和工具的链接。MDN 将 [MathML Core](https://w3c.github.io/mathml-core/) 用作参考规范，但由于不稳定的标准化历史，旧版 MathML 特性可能仍会出现在现有的实现和 Web 内容中。
```

File: files/zh-cn/web/mathml/index.md [hunk 8]
```diff
@@ -19,24 +27,19 @@ slug: Web/MathML
   - : 可以让你理解如何使用 MathML 的例子。
 - [编写 MathML](/zh-CN/docs/Web/MathML/Authoring)
   - : 编写 MathML 的一些建议和提示，包括建议使用的编辑器以及如何将编辑器的输出集成到网页中去。
-
-[查看全部...](/zh-CN/docs/tag/MathML)
+- [MathML 教程](/zh-CN/docs/Learn/MathML)
+  - : 一个对 MathML 的简要介绍。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/index.md:31]:
```suggestion
  - : 对 MathML 的简要介绍。
```

**Review by yin1999 — COMMENTED** [review]
nits

**Review by yin1999 — APPROVED** [review]

**Pull request merged**